.github/workflows/main.yml

```diff
@@ -1,11 +1,13 @@
 name: Main
```

Comment:
do we want a more meaningful name for the workflow here?

Reply:
Updated